txclib/project.py

```diff
@@ -502,6 +502,9 @@ def pull(self, languages=[], resources=[], overwrite=True, fetchall=False,
                     msg = "Resource %s doesn't exist on the server."
                     logger.error(msg % resource)
                     continue
+                if isinstance(e, ConnectionError) or not skip:
```

Comment:
I don't see why you need to catch and reraise each time. Won't this be automatically handled in https://github.com/transifex/transifex-client/pull/196/files#diff-7976d2ef75f85f33a628bad6981a0e18R124 ?

Reply:
Sure, but I thought that connection errors are just like SSL ones, so they should be treated in the same way. Let's proceed with whatever seems better to you.